CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+- Proc names should be written in CamelCase.
+
+- Constants and preprocessor macros/defines should be written in UPPERCASE.
```

Comment:
I actually meant both, seems like it would be more appropriate than mixing constant vars and constant defines.